src/App.tsx

```diff
@@ -7,6 +7,7 @@ import { HomePage } from './home-page/HomePage';
 import { theme } from './layout/theme';
 import { DetailPage } from './detail-page/DetailPage';
 import { ITProfessions } from './data/ITProfessions';
+import { NoMatch404 } from './home-page/NoMatch404';
```

Comment:
We have folder "src/layout/components" for components which will be used in different places in the page (MainContainer is there). Maybe it will be better place for 404 page component because it is not specific for home page only.

Reply:
That's a good point! Alternatively we could set up a separate folder for error pages. But for now we will only have 404, so it seems an overkill.

Reply:
done